additional-test-app-package.json

```diff
@@ -8,6 +8,5 @@
   "scripts": {
     "test:watch": "ember test --server",
     "test": "npm-run-all lint \"test:!(watch)\""
-  },
-  "private": true
```

Comment:
this does not seem to be needed as it's already part of the ember-cli app blueprint https://github.com/ember-cli/ember-cli/blob/v4.8.0/blueprints/app/files/package.json#L4